Makefile

```diff
@@ -135,6 +135,9 @@ olm-push:
 bump-kubevirtci:
 	./hack/bump-kubevirtci.sh
 
+focus:
```

Comment:
I miss a target to revert this.

Reply:
just make focus without params reverts, see please the md file changes
i can alias it if need as unfocus

Reply:
Thanks